app/src/main/java/co/nimblehq/compose/crypto/ui/components/linechart/GradientLineShader.kt

```diff
@@ -0,0 +1,19 @@
+package co.nimblehq.compose.crypto.ui.components.linechart
+
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.Canvas
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.graphics.drawscope.DrawScope
+import me.bytebeats.views.charts.line.render.line.ILineShader
+
+data class GradientLineShader(
+    val colors: List<Color> = listOf(Color.Cyan, Color.Transparent)
+) : ILineShader {
+    override fun fillLine(drawScope: DrawScope, canvas: Canvas, fillPath: Path) {
+        val gradient = Brush.verticalGradient(
+            colors = colors
+        )
+        drawScope.drawPath(path = fillPath, brush = gradient)
+    }
+}
```

Comment:
Why don't we use the original [GradientLineShader](https://github.com/bytebeats/compose-charts/blob/master/charts/src/main/java/me/bytebeats/views/charts/line/render/line/GradientLineShader.kt)?

![image](https://user-images.githubusercontent.com/16315358/195394376-c4a43977-581a-4a45-b621-8a8e3f284163.png)

Reply:
Ops. Seems I updated something and then reverted it back. We could use the Shader from library